src/table.rs

```diff
@@ -2074,8 +2088,19 @@ impl<'a, T> Iterator for IterHash<'a, T> {
             None => None,
         }
     }
+
+    fn fold<B, F>(self, init: B, mut f: F) -> B
+    where
+        Self: Sized,
+        F: FnMut(B, Self::Item) -> B,
+    {
+        self.inner
+            .fold(init, |acc, bucket| unsafe { f(acc, bucket.as_ref()) })
```

Comment:
This is *okay*, but the inner `RawIterHash` isn't doing anything special for `fold`.
(Ditto for `IterHashMut`, and `Drain`'s `RawDrain`.)